utils.py

```diff
@@ -226,63 +225,12 @@ def convert_examples_to_features(examples, label_list, max_seq_length,
 
     label_map = {label : i for i, label in enumerate(label_list)}
 
-    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id) for example in examples]
+    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id, sep_token_extra) for example in examples]
```

Comment:
Some changes existed in one of the `convert_examples_to_features`, some changes existed in the other.

I merged the changes:
* Adding `sep_token_extra=False` arg
* Passing `sep_token_extra` to in `examples` list.
* Adding `process_count=cpu_count() - 2` arg
* `chunksize = 500`